api/src/main/java/mcoffin/rogue/EncoderControl.java

```diff
@@ -0,0 +1,4 @@
+public interface EncoderDrive {
```

Comment:
You're going to need package statements again. You know, you should add a `build.gradle` so you can actually be compiling this as you go / generate an eclipse project.